tests/unittests/media/server/gstplayer/genericPlayer/GstCapabilitiesTest.cpp

```diff
@@ -92,7 +95,17 @@ class GstCapabilitiesTest : public testing::Test
     {
         IFactoryAccessor::instance().gstWrapperFactory() = nullptr;
         IFactoryAccessor::instance().glibWrapperFactory() = nullptr;
-        gst_object_unref(m_elementFactory);
+        IFactoryAccessor::instance().rdkGstreamerUtilsWrapperFactory() = nullptr;
+        if (m_elementFactory)
+        {
+            GstObject *parent = gst_object_get_parent(GST_OBJECT(m_elementFactory));
```

Comment:
This getting and unreffing the parent is redundant code. We shouldn't need to do it.